app/routes.py

```diff
@@ -0,0 +1,269 @@
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+# --------------------------------
+# ----------- CUSTOMERS ----------
+# --------------------------------
+
+
+# GET ALL CUSTOMERS
+@customers_bp.route("", methods=["GET"])
+def get_customers():
+    customers = Customer.query.all()
+    customers_response = []
+    for customer in customers:
+        customers_response.append(customer.to_json())
+    return jsonify(customers_response), 200
+
+
+# CREATE CUSTOMER
+@customers_bp.route("", methods=["POST"])
+def create_customer():
+    request_body = request.get_json()
+    if "name" not in request_body:
+        return jsonify({"details": "Request body must include name."}), 400
+    elif "postal_code" not in request_body:
+        return jsonify({"details": "Request body must include postal_code."}), 400
+    elif "phone" not in request_body:
+        return jsonify({"details": "Request body must include phone."}), 400
+    new_customer = Customer(name=request_body["name"],
+                            postal_code=request_body["postal_code"],
+                            phone=request_body["phone"])
+    db.session.add(new_customer)
+    db.session.commit()
+    return jsonify(new_customer.to_json()), 201
+
+
+# GET CUSTOMER BY ID
+@customers_bp.route("/<id>", methods=["GET"])
+def get_customer(id):
+    if id.isnumeric() != True:
+        return {"message": "Customer id provided is not a number."}, 400
+    customer = Customer.query.get(id)
+    if customer is None:
+        return {"message": f"Customer {id} was not found"}, 404
+    else:
+        return jsonify(customer.to_json()), 200
+
+
+# UPDATE CUSTOMER BY ID
+@customers_bp.route("/<id>", methods=["PUT"])
+def update_customer(id):
+    customer = Customer.query.get(id)
+    if customer is None:
+        return {"message": f"Customer {id} was not found"}, 404
+    if request.method == "PUT":
+        request_body = request.get_json()
+        if "name" not in request_body:
+            return {"details": "Request body must include name."}, 400
+        elif "postal_code" not in request_body:
+            return {"details": "Request body must include postal_code."}, 400
+        elif "phone" not in request_body:
+            return {"details": "Request body must include phone."}, 400
+    customer.name = request_body["name"]
+    customer.postal_code = request_body["postal_code"]
+    customer.phone = request_body["phone"]
+    db.session.commit()
+    return jsonify(customer.to_json()), 200
+
+
+# DELETE CUSTOMER BY ID
+@customers_bp.route("/<id>", methods=["DELETE"])
+def delete_customer(id):
+    customer = Customer.query.get(id)
+    if customer is None:
+        return jsonify({"message": (f"Customer {id} was not found")}), 404
+    else:
+        db.session.delete(customer)
+        db.session.commit()
+        return jsonify({"id": customer.id}), 200
+
+# --------------------------------
+# ----------- VIDEOS ----------
+# --------------------------------
```

Comment:
The basic CRUD operations for videos are largely the same as for customers, so look for places where the feedback on customer endpoints could apply to video endpoints as well.